eng/restore/dirs.proj

```diff
@@ -7,6 +7,7 @@
 
   <!-- Build for all configurations -->
   <ItemGroup>
+    <Project Condition="'$(DotNetBuildFromSource)' != 'true'" Include="analyzers/analyzers.depproj" />
```

Comment:
the comment above isn't correct anymore. I would move it down below this line.

Reply:
Sounds good.